### PR TITLE
Submitting two React Native talks

### PR DESCRIPTION
**Pull request by seidtgeist**
Poof.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,26 +76,24 @@
             </div>
             -->
             <div class="speaker empty">
```
Comment (janmonschke) [inline index.html:78]: Should probably not be `empty`

**Comment by rmehner** [comment]
Is the third talk also coming? If not we've got three interesting other talks lined up that we could use :)

**Comment by rmehner** [comment]
And thanks, looking forward!

**Comment by seidtgeist** [comment]
@rmehner Pinged @davidgruebl

**Comment by rmehner** [comment]
@davidgruebl _ping_ :)

**Comment by seidtgeist** [comment]
Sorry, he already let me know he won't be able to prepare something for this time.

**Pull request merged**